include/exadg/operators/variable_coefficients.h

```diff
@@ -23,144 +23,164 @@
 
 namespace ExaDG
 {
-template<int dim, typename Number>
+template<typename coefficient_t>
```

Comment:
I would like to be explicit about `t` (I guess you mean `coefficient_type`; but one could potentially also think of `coefficient_template`, so I think being explicit helps in general for code readability or when talking about the code).

Reply:
In the C++ language, I do not think that the trailing word `_t` can be misunderstood, because it is part of the language specification, see here for one example: https://en.cppreference.com/w/cpp/language/types - it is just that @btemuer likely did not know all the project's habits. And to be explicit on our habits (that are different from the C++ standard), I agree that spelling out `t` does help, so :+1: from my end on the suggested change of `_type`.